include/nbl/ui/CWindowWin32.h

```diff
@@ -18,6 +18,8 @@ class CWindowWin32 final : public IWindowWin32
 
 public:
 	static LRESULT CALLBACK WndProc(HWND hWnd, UINT message, WPARAM wParam, LPARAM lParam);
+	static E_KEY_CODE getNablaKeyCodeFromNative(uint32_t nativeWindowsKeyCode);
```

Comment:
what is a `nativeWindowsKeyCode` ?

Reply:
https://docs.microsoft.com/en-us/windows/win32/inputdev/virtual-key-codes

Reply:
then why a `uint32_t` and not `uint8_t`

Reply:
done